v2.0/partitioning.md

```diff
@@ -0,0 +1,371 @@
+$ cockroach zone rm <database.table.partition> <flags>
+~~~
+
+### Verify Table Partitions
+
+To verify the table partitions, use the [`SHOW CREATE TABLE`](show-create-table.html) statement.
+
+## Scenario-based Examples
+
+### Define Table Partitions by List
+
+Consider a global online learning portal, RoachLearn, that has a database containing a table of students across the world. Suppose we have two datacenters: one in the United States and another in Australia. To reduce latency, we want to keep the students' data closer to their locations: 
+
+- We want to keep the data of the students located in the United States and Canada in the United States datacenter.
+- We want to keep the data of students located in Australia and New Zealand in the Australian datacenter. 
+
+We can achieve this by partitioning on `country` and using the `PARTITION BY LIST` syntax. 
+
+#### Step 1: Start each node with its datacenter location specified in the `--locality` flag:
+
+~~~ shell
+# Start the node in the US datacenter:
+$ cockroach start --insecure --locality=region=us-1  --store=node1 --host=<node1 hostname> --port=26257 --http-port=8080 --join=<node1 hostname>:26257,<node2 hostname>:26258
+
+# Start the node in the AUS datacenter:
+$ cockroach start --insecure --locality=region=aus-1 --store=node2 --host=<node2 hostname> --port=26258 --http-port=8081 --join=<node1 hostname>:26257,<node2 hostname>:26258
+~~~
+
+#### Step 2: Set the enterprise license
+
+To set the enterprise license, see [Set the Trial or Enterprise License Key](enterprise-licensing.html#set-the-trial-or-enterprise-license-key)
+
+#### Step 3: Create a table with the appropriate partitions:
+
+~~~ sql
+> CREATE TABLE students_by_list (
+    id SERIAL,
+    name STRING,
+    email STRING,
+    country STRING,
+    expected_graduation_date DATE,   
+    PRIMARY KEY (country, id))
+    PARTITION BY LIST (country)
+      (PARTITION north_america VALUES IN ('CA','US'), 
+      PARTITION australia VALUES IN ('AU','NZ'), 
+      PARTITION DEFAULT VALUES IN (default));
+~~~
+
+#### Step 4: Create corresponding zone configurations:
+
+~~~ shell
+$ cat > north_america.zone.yml
+constraints: [+datacenter=us1]
+
+$ cat > australia.zone.yml
+constraints: [+datacenter=au1]
+~~~
+
+#### Step 5: Apply zone configurations to corresponding partitions:
+ 
+~~~ shell
+$ cockroach zone set roachlearn.students_by_list.north_america --insecure  -f north_america.zone.yml
+$ cockroach zone set roachlearn.students_by_list.australia --insecure  -f australia.zone.yml
+~~~
+
+### Define Table Partitions by Range
+
+Consider the table of students of the global online learning portal, RoachLearn. Suppose we want to store the data of current students on fast and expensive storage devices (example: SSD) and store the data of the graduated students on slower, cheaper storage devices(example: HDD). We can achieve this by partitioning the table by date and using the `PARTITION BY RANGE` syntax. 
```

Comment:
I don't think you need the first sentence.

nits: 
- `(example: SSD)` > `(e.g., SSD)`
- `(example: HDD)` > `(e.g., HDD)`

Also need a space before the HDD parentheses.